index.bs

```diff
@@ -12032,22 +12034,18 @@ defined in section [[#platformobjectset]].
 </div>
 
 
-<h4 id="getownproperty">Platform object \[[GetOwnProperty]] method</h4>
+<h4 id="legacy-platform-object-getownproperty" oldis="getownproperty">Legacy platform object \[[GetOwnProperty]] method</h4>
```

Comment:
…and in the process of removing them, I found out why I had added them in the first place: one of the ids is: "platformobjectset" which is incorrect, so then decided to fix the whole bunch for consistency. I'll leave them until someone else complains. :D